pkg/acsengine/const.go

```diff
@@ -123,7 +123,7 @@ var KubeConfigs = map[string]map[string]string{
 		"gclowthreshold":  strconv.Itoa(DefaultKubernetesGCLowThreshold),
 	},
 	api.KubernetesVersion1Dot7Dot7: {
-		"hyperkube":       "hyperkube-amd64:v1.7.7",
+		"hyperkube":       "hyperkube-amd64:v1.7.8",
```

Comment:
zip file update needed.

Reply:
@JiangtianLi please let us know when we have a `v1.7.8` windows artifact